buildconfig/pygame-stubs/rect.pyi

```diff
@@ -181,6 +181,16 @@ class Rect(object):
     def fit(self, left: float, top: float, width: float, height: float) -> Rect: ...
     def normalize(self) -> None: ...
     @overload
+    def __contains__(self, rect: Union[_RectStyle, Rect, int]) -> bool: ...
+    @overload
+    def __contains__(
+        self,
+        left_top: Union[List[float], Tuple[float, float], Vector2],
+        width_height: Union[List[float], Tuple[float, float], Vector2],
+    ) -> bool: ...
```

Comment:
Ah, seems like there's a merge conflict in here, sorry it happened because one of my typehint fix PRs got merged. To fix this on this PR, replace any `_RectStyle` with `_CanBeRect`

Also `Union[List[float], Tuple[float, float], Vector2]` can be replaced with `_Coordinate` which is imported from `_common` typestub file

Reply:
Done